azurerm/resource_arm_automation_account.go

```diff
@@ -221,9 +262,21 @@ func flattenAutomationAccountSku(sku *automation.Sku) []interface{} {
 }
 
 func expandAutomationAccountSku(d *schema.ResourceData) *automation.Sku {
-	inputs := d.Get("sku").([]interface{})
-	input := inputs[0].(map[string]interface{})
-	name := automation.SkuNameEnum(input["name"].(string))
+	v := d.Get("sku_name").(string)
```

Comment:
Can we move this logic into the main function as we'll remove this in 2.0 and it'll end up being

sku = automation.SkuNameEnum(d.Get("sku_name").(string))

Reply:
Move the logic and removed the function